wagtail/wagtailadmin/tests/test_pages_views.py

```diff
@@ -2029,6 +2029,99 @@ def test_unpublish_view_post(self):
         self.assertEqual(mock_call['instance'], self.page)
         self.assertIsInstance(mock_call['instance'], self.page.specific_class)
 
+    def test_unpublish_descendants_view(self):
+        """
+        This tests that the unpublish view responds with an unpublish confirm page that contains the form field 'include_descendants'
```

Comment:
This docstring says the page must contain the checkbox, while the test checks the page does *not* contain the checkbox.